modules/nixos/common/default.nix

```diff
@@ -33,6 +33,7 @@
   ) pkgs.linuxPackages_latest;
 
   zramSwap.enable = true;
+  zramSwap.memoryPercent = 100;
```

Comment:
@Mic92 Default is 50, I don't think there is any problem with increasing this on all hosts?